pkg/cnab/provider/list.go

```diff
@@ -0,0 +1,60 @@
+package cnabprovider
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+
+	printer "github.com/deislabs/porter/pkg/printer"
+)
+
+// CondensedClaim holds a subset of pertinent values to be listed from a claim.Claim
+type CondensedClaim struct {
+	Name     string
+	Created  time.Time
+	Modified time.Time
+	Action   string
+	Status   string
+}
+
+// List lists bundles with the printer.Format provided
+func (d *Duffle) List(opts printer.PrintOptions) error {
+	claimStore := d.NewClaimStore()
+	claims, err := claimStore.ReadAll()
+	if err != nil {
+		return errors.Wrap(err, "could not list claims")
+	}
+
+	var condensedClaims []CondensedClaim
+	for _, claim := range claims {
+		condensedClaim := CondensedClaim{
+			Name:     claim.Name,
+			Created:  claim.Created,
+			Modified: claim.Modified,
+			Action:   claim.Result.Action,
+			Status:   claim.Result.Status,
+		}
+		condensedClaims = append(condensedClaims, condensedClaim)
+	}
+
+	switch opts.Format {
+	case printer.FormatJson:
+		return printer.PrintJson(d.Out, condensedClaims)
+	case printer.FormatYaml:
+		return printer.PrintYaml(d.Out, condensedClaims)
+	case printer.FormatTable:
+		printClaimRow :=
+			func(v interface{}) []interface{} {
+				cl, ok := v.(CondensedClaim)
+				if !ok {
+					return nil
+				}
+				return []interface{}{cl.Name, cl.Created, cl.Modified, cl.Action, cl.Status}
```

Comment:
It would be great to either in this PR or in a follow-up format the dates in a human friendly format. I'm cool with either finding a go format that looks less verbose or using a package like go humanize.

Reply:
Updated with a friendly format; let me know what you think.